src/main/java/de/muenchen/allg/itd51/wollmux/event/handlers/OnProcessTextDocument.java

```diff
@@ -22,24 +24,18 @@
 public class OnProcessTextDocument extends WollMuxEvent
 {
   private static final Logger LOGGER = LoggerFactory.getLogger(OnProcessTextDocument.class);
-
   private TextDocumentController documentController;
 
-  private boolean visible;
-
-  /**
-   * Create this event.
-   *
-   * @param documentController
-   *          The document to process.
-   * @param visible
-   *          If false, the window of the document is invisible, otherwise it's visible.
-   */
-  public OnProcessTextDocument(TextDocumentController documentController,
-      boolean visible)
+  public OnProcessTextDocument()
```

Comment:
EventListener und Event in einem finde ich nicht schön. Kannst du das bitte noch in dem allgemeinen EventListener lassen. Dann ist auch der Aufruf von `new OnProcessTextDocument` im LibreOfficeEventListener nicht notwendig